### DISCUSSION TITLE
我无法验证学生认证

### DISCUSSION THREAD

**Discussion by 12362566** [community/community#55248] [GitHub Education]
### Hi everyone,

You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.

**Comment by samus-aran** [comment]
Hi @12362566 

For remote student applicants: please ensure that your evidence indicates that you are a remote student. If possible, we encourage students to apply whilst on campus. This will assist us in confirming your student status.

Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

Hope this helps 👍
